Extension/src/githubAPI.ts

```diff
@@ -9,13 +9,14 @@ import * as util from './common';
 import { PlatformInformation } from './platform';
 import { OutgoingHttpHeaders } from 'http';
 import * as vscode from 'vscode';
+import * as telemetry from './telemetry';
 
 const testingInsidersVsixInstall: boolean = false; // Change this to true to enable testing of the Insiders vsix installation.
```

Comment:
@sean-mcmanus do we need this variable anymore?

Reply:
Maybe. Might be useful for manual testing/debugging of the code to force an install.